### PR TITLE
Ch5117/from ibx match list responses not saving

### PR DESCRIPTION
**Pull request by elodszopos**

### CODE REVIEW

**Comment by elodszopos** [comment]
Solves [ch5117]

File: packages/extended-text-entry/src/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ export default class RootExtendedTextEntry extends HTMLElement {
     this._session.value = value;
 
     this.dispatchEvent(
-      new SessionChangedEvent(this.tagName.toLowerCase(), this._session)
+      new SessionChangedEvent(this.tagName.toLowerCase(), true)
```
Comment (edeustace) [inline packages/extended-text-entry/src/index.js]: always complete? maybe check if  it's a non empty string instead?

File: packages/match-list/src/index.js [hunk 2]
```diff
@@ -13,6 +15,19 @@ export default class MatchList extends HTMLElement {
     super();
   }
 
+  isComplete() {
```
Comment (edeustace) [inline packages/match-list/src/index.js]: test

**Comment by elodszopos** [comment]
Updated @edeustace

**Pull request merged**